### DISCUSSION TITLE
Uncaught (in promise) TypeError: i is not a function

### DISCUSSION THREAD

**Discussion by ricardo118** [alpinejs/alpine#1992] [7. Bugs (archived)]
Hello, 

Ive been using alpine v2 for a long time now and decided to try a project in v3. Its still very early on but I have a 
```
<body id="top" class="text-black bg-repeat-round"
      x-data="{ageGate: true, menu: true}" 
      x-init="ageGate = localStorage.getItem('ageGate')!=='false'">

...

    <a href="" class="" @click.prevent="menu = !menu">
        <div class="" x-show="!menu">
            <svg>...</svg>
        </div>
        <div class="" x-show="menu" >
           <svg>...</svg>
        </div>
    </a>
```

its a very simple toggle state that i've used multiple times, but it prints `Uncaught (in promise) TypeError: i is not a function` exactly 16 times everytime you click. (8 per x-show i assume. 8 for the `menu` and another 8 for the `!menu`.

The line it complains about is like 1713 of cdn.js "version": "3.2.4"

Anything i'm doing wrong for v3? I tried the docs example but had the same result.

Thank you

**Comment by ricardo118** [comment]
Fixed, unfortunately I had forgotten the `defer` attr on the <script></script> tag when importing alpine.js

Leaving this here for future references.

Solution:
Make sure the `defer` attribute is present.

![image](https://user-images.githubusercontent.com/7531933/131159794-8d44d0a8-fac7-4a21-9911-0e3dc2da34d0.png)

**Reply by rliessum** [reply]
Defer is one, or if you import the lib twice... same issue. Just so you know if you run into it.
